Lombiq.HelpfulLibraries.OrchardCore/Liquid/ILiquidParserTag.cs

```diff
@@ -0,0 +1,23 @@
+using Fluid;
+using Fluid.Ast;
+using System.Collections.Generic;
+using System.IO;
+using System.Text.Encodings.Web;
+using System.Threading.Tasks;
+
+namespace Lombiq.HelpfulLibraries.OrchardCore.Liquid;
+
+/// <summary>
+/// Describes a Liquid parser tag's parser and render methods.
```

Comment:
It's unclear what a "Liquid parser tag" is. Please elaborate in the comment.

Reply:
It's a "parser tag" in Liquid. :D I don't know, it's just the term [OC uses](https://github.com/OrchardCMS/OrchardCore/blob/51f24d72211f8642976547e34aef018e05b941fa/src/OrchardCore/OrchardCore.DisplayManagement.Liquid/LiquidViewParser.cs#L46) for the the other things that aren't filters.

Reply:
I see, so it's an OC technical term. I thought it's something about a tag that parses Liquid as its input as opposed to literals. Nothing needed, then.